draft-ietf-idr-deprecate-as-set-confed-set.xml

```diff
@@ -268,10 +245,13 @@ Route aggregation that was previously performed by proxy aggregation without the
           include the ATOMIC_AGGREGATE attribute.</t>
         </blockquote>
 
-        <t>When BGP AS_PATH aggregation is done according to the Section 9.2.2.2
+        <t>When BGP AS_PATH aggregation is done according to the <xref target="RFC4271" section="9.2.2.2" sectionFormat="comma"/>,
         procedures and any resulting AS_SETs are discarded, this is typically
-        referred to as "brief" aggreation in implementations.  This results in
-        an AS_PATH that has the property (from Section 9.2.2.2):</t>
+        referred to as "brief" aggregation in implementations.
+(Note: It is true in practice and assumed here that AS_SETs 
+occur only at the beginning (origination side) of the AS_PATHs.)

```

Comment:
I would skip this parenthetical (and mis-indented) statement.  It doesn't necessarily help with the explanation, and the "beginning" portion isn't how we'd normally refer to this in RFC land.

Reply:
done